layers/+tools/eaf/packages.el

```diff
@@ -236,20 +235,19 @@
         (lambda (prompt)
           (if (derived-mode-p 'eaf-mode)
               (pcase eaf--buffer-app-name
-                ((or
-                  (and "browser"
-                       (guard (string= (eaf-call-sync "call_function" eaf--buffer-id "is_focus") "True")))
-                  "image-viewer"
-                  "pdf-viewer")
-                 (kbd eaf-evil-leader-key))
+                ("browser" (if (string= (eaf-call-sync "call_function" eaf--buffer-id "is_focus") "True")
+                               (kbd "SPC")
+                             (kbd eaf-evil-leader-key)))
+                ("pdf-viewer" (kbd eaf-evil-leader-key))
+                ("image-viewer" (kbd eaf-evil-leader-key))
                 (_  (kbd "SPC")))
             (kbd "SPC"))))
 
       ;; The following lines create the major-mode leader key emulation map
       ;; in a similar way as how it was done in the evil-integration example
       (setq eaf-evil-leader-for-major-keymap (make-sparse-keymap))
       (define-key eaf-evil-leader-for-major-keymap (kbd "h") 'eaf-open-browser-with-history)
-      (define-key eaf-evil-leader-for-major-keymap (kbd "d") 'eaf-toggle-dark-mode)
```

Comment:
This function is defined in `funcs.el` and now it's not used.
Do we need to remove it? But it works 100% of time on my computer....

Reply:
Are you sure that function works, because here it did not. I did not remove it, but replace it with the working version of that function (see below)

Reply:
I can only view PDF file in EAF on my computer. And the only working function is to toggle dark mode. After this commit that stopped working.

Reply:
No, the it should not be this line.

Reply:
What I meant is https://github.com/syl20bnr/spacemacs/blob/df15f8cc159b9fe4380144355af077935e7770b7/layers/%2Btools/eaf/funcs.el#L34

That is not used after this PR. And that function works for me.

Reply:
Haha okay, I forgot that I had added that function myself. But I am using mainly the eaf-browser, and there this function stopped working while that new function does work. So I guess maybe we require both functions. I will look into it a little more then.